### ISSUE TITLE
issues with ng-show?

### ISSUE THREAD

**Issue by Ir1sh** [fergaldoyle/angular.js-ie8-builds#5]
Have you guys noticed any issues with ng-show working in ie8? I cant get it to work without overriding the default style as described int he angular source here

By default, the `.ng-hide` class will style the element with `display: none!important`. If you wish to change
 the hide behavior with ngShow/ngHide then this can be achieved by restating the styles for the `.ng-hide`
 class in CSS:
 
  ```css
  .ng-hide {
    /&#42; this is just another form of hiding an element &#42;/
    display: block!important;
    position: absolute;
    top: -9999px;
    left: -9999px;
  }
  ```
not sure if and how this will effect animations, looking into it now

**Comment by fergaldoyle** [comment]
Hadn't noticed that before.

Probably the cleanest thing would be to include in the IE8 conditional the css for ng-hide.

    

Since by default Angular injects `.ng-hide:not(.ng-hide-animate){display:none !important;}`  which IE8 won't understand.

**Comment by fergaldoyle** [comment]
I just released v1.3.13 which will automatically inject `.ng-hide {display: none !important;}` for IE8 only. It won't break animations (CSS based ones won't work in IE8 anyway, JavaScript animations work fine)
